CHANGELOG.md

```diff
@@ -1,20 +1,33 @@
 # Dunst changelog
 
-## Unreleased
+## 1.3.0 - 2017-12-05
 
 ### Added
-- `ellipsize` option to control how long lines should be ellipsized when `word_wrap` is set to `false`
+- `ellipsize` option to control how long lines should be ellipsized when `word_wrap` is set to `false` (#374)
+- A beginning tilde of a path is now expanded to the home of the current user (#351)
+- The image-path hint is now respected, as GApplications send their icon only via this link (#447)
+- If dunst can't acquire the DBus name, dunst prints the PID of the process, holding the name (#458 #460)
```

Comment:
The second comma on this line shouldn't be there.

Reply:
Thank you @WhyNotHugo